lib/vtls/openssl.c

```diff
@@ -1115,6 +1120,10 @@ static int x509_name_oneline(X509_NAME *a, char *buf, size_t size)
  */
 static int Curl_ossl_init(void)
 {
+#if (OPENSSL_VERSION_NUMBER >= 0x10100000L) &&  \
+  !defined(LIBRESSL_VERSION_NUMBER)
+  OPENSSL_init_ssl(OPENSSL_INIT_ENGINE_ALL_BUILTIN | INITFLAGS, NULL);
```

Comment:
Fine with me. Consider moving all of it here, for example I think this is faster to understand
```suggestion
  uint64_t flags = OPENSSL_INIT_ENGINE_ALL_BUILTIN |
#ifdef CURL_DISABLE_OPENSSL_AUTO_LOAD_CONFIG
                   OPENSSL_INIT_NO_LOAD_CONFIG;
#else
                   OPENSSL_INIT_LOAD_CONFIG;
#endif
  OPENSSL_init_ssl(flags, NULL);
```

Reply:
Seems fine to me, and do it `const`.